### ISSUE TITLE
Update Mainland map to PreRremaster tweaked map.

### ISSUE THREAD

**Issue by zajako** [zajako/LineageHD#276]

**Comment by AeacosL15** [comment]
A lot of new NPCs that don't seem to talk to you.

I went through the -warp locations ... old NPCs seem to mostly be good ... only key ones I noticed missing were the Alchemist (in SKT, Giran, Weldern towns), but there is one in Oren town.

All the Ancient Orcs and Ancient Beleth seemed to be missing from the orc forest (west of -warp 6).

The wasteland (around chaotic temple, north of Gludio) has the original mobs, not the newer ancient mobs.

There used to be an NPC Giant near -warp 9 (the open area where Ancient Giant spawns) that is missing, but don't remember what he was for.

Some teleport NPCs seem to open 2 teleport windows.  The old style which doesn't work and the new style.

NPCs in Gypsy Town are missing.  You can't go to Dream Island.  (Correction - NPC to Dream Island has just moved to the South East of where it used to be.)

Lindvior spawn area is missing, so I'm not sure if he's still there.

Fire Valley and it's castle seem to be missing/blocked off (no Valakas access?).

SKT Caves seem to be inaccessible even from old bookmarks.

Dragon Valley - Drakes/DE still spawn but nothing else.  Cave entrances don't seem to be there anymore (can use teleport NPC to get to caves for Antharas).  Old bookmarks work, but I'm unable to move.

Ant Caves - Entrances seem to still be there, they're just not graphically visible.

**Comment by zajako** [comment]
I fixed the mob spawns. 

I believe I've fixed the 2 teleport windows.

After restart eldin will be moved back up a bit in gypsy town

Lindvior's Scar is a bit further north from the old spawn location, (After restart it will spawn up there too)

Fire Valley was Destroyed by Valakas, Dragon Valley was destroyed by Antharas the old npcs have been moved to the furthest north point in between Dragon Valley and Fire Valley.  Those npcs can teleport you to fight the dragons.

Ant Caves needs a new ticket.
